_data/api-swagger.yaml

```diff
@@ -472,12 +475,18 @@ definitions:
           - Kennisnet
           - PhoneNumber
           - Scribble
-          - SigningCertificate 
+          - SigningCertificate
           - SURFnet
-        description:
+        description: |
           Type of the verification object.
           The `Type` property **must** be the first property in the json!
 
+          The order in which the verifications are provided determine in which order the signer will have to perform the specified method. You **must** use one of the following verifications as the last method:
```

Comment:
Please place each sentence on a new line to reduce merge conflict in the future.